_posts/blog/2018-10-10/2018-10-10-how-to-buy-cryptocurrencies-privately-with-no-fees.markdown

```diff
@@ -0,0 +1,86 @@
+---
+title: "How to Buy Cryptocurrencies <span class='post-title-extra'>Privately with no Fees</span>"
+layout: post
+date: '2018-10-10 06:30:00 -0300'
+social_title: 'How to Buy Cryptocurrencies Privately with no Fees'
+social_description: 'This tutorial explains how to use OpenBazaar to purchase cryptocurrencies.'
+---
+
+This tutorial will explain how to purchase cryptocurrencies on OpenBazaar, a decentralized marketplace which never has fees and doesn’t require opening an account. This is for people buying cryptocurrency with a different cryptocurrency, such as Bitcoin. At the moment OpenBazaar doesn’t support people buying cryptocurrency with regular fiat money such as US Dollars or Euros, though it’s a feature that will hopefully be supported in the future.
```

Comment:
> This is for people buying cryptocurrency with a different cryptocurrency, such as Bitcoin.

This sounds kind of weird, because the subject of the "this" is unclear, it sounds like it might be referring to "OpenBazaar" or "opening an account" vs. "This tutorial".

Maybe "You'll learn how to buy a cryptocurrency with a different..."?